src/Microsoft.Health.Fhir.Tests.Common/TestFiles/R5/AuthConfigWithInvalidEntries.json

```diff
@@ -0,0 +1,40 @@
+﻿{
```

Comment:
Should make an issue to consolidate these Auth files. Doesn't seem like they would change with version

Reply:
These were removed in R4 and Stu3, not sure if it was recent. I'll remove them from R5.